docs/how-to-guides/faq.md

```diff
@@ -74,4 +74,24 @@ Below is the serving logs after setting `PF_LOGGING_LEVEL` to `DEBUG`:
 
 Compare to the serving logs with `WARNING` level:
 
-![img](../media/how-to-guides/pf_logging_level_warning.png)
\ No newline at end of file
+![img](../media/how-to-guides/pf_logging_level_warning.png)
+
+### Set environment variables
+
+Currently, promptflow supports the following environment variables:
+
+**PF_WORKER_COUNT** 
+Valid for batch run only. The number of workers to use for parallel execution of the Flow.
+
+**PF_BATCH_METHOD**
+Valid for batch run only. Optional values: 'spawn', 'fork', 'forkserver'. [Details](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.AsyncResult:~:text=Depending%20on%20the%20platform%2C%20multiprocessing%20supports%20three%20ways%20to%20start%20a%20process.)
+
+#### You can configure environment variables in the following ways
+1. If you are using the SDK, you can use this parameter: ```--environment-variable```. Example: ```--environment-variable key1="\`${my_connection.api_key}\`" key2="value2"```.
```

Comment:
do you mean cli here?

Reply:
Yes, I have corrected it and added the SDK example.